src/main/java/org/mds/pacman/Direction.java

```diff
@@ -1,20 +1,60 @@
 package org.mds.pacman;
 
-import javax.swing.*;
-import java.awt.*;
+import javax.swing.ImageIcon;
+import java.awt.Image;
+
+import static org.mds.pacman.Paths.pacman;
+import static org.mds.pacman.Paths.pacmanDown;
+import static org.mds.pacman.Paths.pacmanDown2;
+import static org.mds.pacman.Paths.pacmanLeft;
+import static org.mds.pacman.Paths.pacmanLeft2;
+import static org.mds.pacman.Paths.pacmanRight;
+import static org.mds.pacman.Paths.pacmanRight2;
+import static org.mds.pacman.Paths.pacmanUp;
+import static org.mds.pacman.Paths.pacmanUp2;
 

```

Comment:
We will need to change this class, extract the animation logic for sprites and pacman.